rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -56,7 +70,9 @@ def get_filtered_results(self, start_date, end_date, filters, stratification='EN
         values = result[:-1]
         for i, value in enumerate(values):
           key = strata[i]
-          if value == None:
+          if value is None or (query_for_enrollment_status
```

Comment:
If we're going to support not getting all enrollment statuses back, then we probably shouldn't run the SQL necessary to calculate them, or populate anything in the response for them.

But I think it's OK for us to always return all enrollment statuses?

Reply:
In the new sql, we depend on the register time, member time and core time, so for some case even the enrollment_status is 3, but it's still in member status for that query date. In this case when we query for core participants, since for that query date it's still in member status, so there will not be 0 in the member count in the result. That's why I need to check it here.